docs/content/Coding Milestones/PoC2023q1/example1-subs/example1-post-kcp.md

```diff
@@ -92,11 +92,66 @@ to pass the new file with `--kubeconfig` on the command lines when
 using kcp or KubeStellar.
 
 
+### Space Manager
+
+The Space Manager will use space provider to create and manage underlying cluster
+for each Space object. Space objects are created in `root:space-mgt` context
+under `spaceprovider-default` namespace.
+
+```shell
+space-manager --context root:space-mgt &
```

Comment:
Is this configuring the space-manager to use a pre-existing context or telling it to create one?  I suspect the former.  If so, the reader of this document has not seen what causes that context to exist (unless that appears earlier due to a file that GitHub is listing later in this PR).

Reply:
This kubeconfig context is created on `kubestellar init`. Should we document this in `Initialize the KubeStellar platform` section of this document or/and somewhere else?